ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-release-4.14.yaml

```diff
@@ -0,0 +1,111 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: lifecycle-agent-operator
+- dockerfile_path: bundle.Dockerfile
+  to: lifecycle-agent-operator-bundle
+operator:
+  bundles:
+  - dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift-kni/lifecycle-agent-operator:.*
+    with: pipeline:lifecycle-agent-operator
+promotion:
+  to:
+  - name: "4.14"
+    namespace: ocp-kni
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: |
+  ./hack/install-markdownlint.sh
+tests:
+- as: ci-job
+  commands: |
+    unset VERSION
+    make ci-job
+  container:
+    from: src
+- always_run: false
+  as: integration
+  skip_if_only_changed: ^docs/|LICENSE|OWNERS|\.md$|^hack$
+  steps:
+    cluster_profile: aws-telco
+    env:
+      BASE_DOMAIN: telco5g-ci.devcluster.openshift.com
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: openshift-lifecycle-agent
+      OO_PACKAGE: lifecycle-agent
+      OO_TARGET_NAMESPACES: '!install'
+      USER_TAGS: |
+        ci-job-source prowci
+        ci-job-branch main
+        ci-job-fullname openshift-kni-lifecycle-agent-main
+        ci-job-type optional-operators-ci-aws
+        ci-repo-name lifecycle-agent
+        ci-org-name openshift-kni
+    test:
+    - as: integration
+      cli: latest
+      commands: |
+        make test
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: optional-operators-ci-aws
+- always_run: false
+  as: markdownlint
+  commands: |
+    hack/markdownlint.sh
+  container:
+    from: test-bin
+  run_if_changed: (\.md|^Makefile|^hack/.*markdown.*|^.markdown.*)$
+- as: ibu-e2e-flow
+  cluster: build02
+  optional: true
+  skip_if_only_changed: ^docs/|LICENSE|OWNERS|\.md$|^hack$
+  steps:
+    cluster_profile: aws
+    dependencies:
+      LCA_PULL_REF: pipeline:lifecycle-agent-operator
+    env:
+      OCP_BASE_VERSION: "4.14"
+      OCP_IMAGE_SOURCE: ci
```

Comment:
I would make this release to test against the latest released image

Reply:
The e2e flow should also probably be added to the 4.15 branch

Reply:
Apologize if I asked about this before, what exactly is this version?

Reply:
`OCP_BASE_VERSION` is the version of OCP that will be used for the seed and target clusters
`OCP_IMAGE_SOURCE` determines where those images are pulled from (`ci` is images built in CI, `release` will use officially publically released images of OCP. I'm of the opinion that for release branches, we should be testing against released OCP images

Reply:
Also, this comment was merely stating that the config file for release-4.15 does not have this e2e workflow in it

Reply:
Updated